GW2EIEvtcParser/EncounterLogic/Raids/W8/UraTheSteamshrieker.cs

```diff
@@ -130,6 +153,7 @@ internal override void EIEvtcParse(ulong gw2Build, EvtcVersionEvent evtcVersion,
         {
             shard.OverrideID(ArcDPSEnums.TrashID.UraGadget_BloodstoneShard);
             shard.OverrideType(AgentItem.AgentType.NPC);
+            shard.OverrideName("Bloodstone Shard");
```

Comment:
Necessary? Generally I try to avoid overriding ArcDPS inputs if avoidable. Any added value for doing this override?

Reply:
Added it so that they wouldn't be all called ura but i can remove it

Reply:
But do their name appear anywhere inside htmls/jsons ? If yes, keep the override, otherwise we don't need it.

Reply:
Dont think so, they should be just the icon as a generic agent in the combat replay

Reply:
Removed it, if something changes for cm we can add it back later